PetShareTests/AnnouncementEndpointTests.cs

```diff
@@ -311,7 +311,8 @@ public async Task GetPaginatedAnnouncementsWithFiltersNonEmpty()
         {
             PageNumber = 0,
             PageCount = 1,
-            MaxAge = 1000
+            MaxAge = 1000,
+            Status = "Open"
```

Comment:
I'm 90% sure there are lowercase statuses in specification. I don't think it's wrong if we don't _require_ them to be lowercase and allow both versions, but we should use lowercase statuses in tests as it is the most important behaviour